packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -713,6 +717,10 @@ function shouldPreventDefault(target: Element) {
   return !target.closest('[draggable="true"]');
 }
 
+function shouldPreventDefaultKeyboard(target: Element) {
+  return !((target.tagName === 'INPUT' || target.tagName === 'BUTTON') && target.getAttribute('type') === 'submit');
```

Comment:
Doesn't the `button` element have `type="submit"` by default?

Reply:
huh, that is the craziest default I've seen
`submit: The button submits the form data to the server. This is the default if the attribute is not specified for buttons associated with a <form>, or if the attribute is an empty or invalid value.`
Why would it ever be submit outside of a form?
thanks for alerting me to that, I'll update it